WKWebViewRTC/Js/src/iosrtc.js

```diff
@@ -87,12 +87,13 @@ module.exports = {
 	mediaStreams:          mediaStreams,
 	nativeCallback:		   exec.nativeCallback
 };
+
+// register global variables right after webview loading
 registerGlobals();
-initAudioDevices();
-turnOnSpeaker(true);
-requestPermission(true, true, function (result) {
-	console.log('requestPermission.result', result);
-	});
+
+MediaStreamTrack.prototype.clone = function () {
```

Comment:
We have clone in iosRTC i think now.

Reply:
This is using GoogleWebRTC from cocoapods.

Reply:
Ok, we did implement real clone lately I think and it's Twillio friendly.

Reply:
See https://github.com/cordova-rtc/cordova-plugin-iosrtc/blob/master/src/iosrtcPlugin.swift#L794